clusters/kepler.yaml

```
@@ -0,0 +1,30 @@
+# Refer: https://artifacthub.io/packages/helm/kepler/kepler?modal=install
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: kepler-repo
+  namespace: kepler  # TODO: fix some better place where to store
+spec:
+  interval: 2m
+  url: https://sustainable-computing-io.github.io/kepler-helm-chart
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: kepler
+  namespace: kepler
+spec:
+  interval: 5m
+  # https://fluxcd.io/flux/components/helm/helmreleases/#install-configuration
+  install:
+    createNamespace: true
+  chart:
+    spec:
+      chart: kepler/kepler
```

```suggestion
      chart: kepler
```

Chart should just be `kepler` the prefix is used by the helm CLI to reference the repo.